### ISSUE TITLE
Do you divide the LAION-Aesthetic-6.5+ dataset into a training set and a test set?

### ISSUE THREAD

**Issue by xuyang527** [luosiallen/latent-consistency-model#98]
I do not see a test set in my LAION dataset, there are only images and their corresponding information in this dataset. How do you creat your test set?

**Comment by XingtongGe** [comment]
I have the same question, may I ask have you solved this problem?

**Comment by hoangdzung** [comment]
I also have the same question. I can't really find the so-called test set to evaluate the FID.
